index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+
+    <meta charset="utf-8">
+    <title>About Me - Day Two</title>
+    <link type="text/css" rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+      <h1>About Me</h1>
```

Comment:
your indentation ins inconsistent, inside your body you are using 4 spaces, but inside your head you are using 2 space.  always use two spaces.  I think atom's tab automatically converts to 2 spaces, so when in doubt just use tab